aequilibrae/parameters.yml

```diff
@@ -198,6 +198,84 @@ network:
         mode_filter:
           pedestrian: 'no'
         unknown_tags: true
+  gmns:
+    critical_dist: 2
+    node_fields:
```

Comment:
node_fields, other_node_fields and required_node_fields.  This looks like a mess and would be absolutely impossible for anyone to understand

Reply:
Changed this in the last commit as well.